tests_integration/spcs/testing_utils/spcs_services_utils.py

```diff
@@ -110,8 +112,24 @@ def logs_should_return_service_logs(
         result = self._execute_logs(service_name, container_name)
         assert result.output
         # Assert this instead of full payload due to log coloring
-        assert service_name in result.output
         assert expected_log in result.output
+        payload = json.loads(result.output)
+        is_valid = self.verify_included_timestamps(payload)
+        assert is_valid == True
```

Comment:
other methods here use direct asserts, let's keep doing that
```suggestion
        self.verify_included_timestamps(payload)
```